content/en/docs/ops/integrations/spire/index.md

```diff
@@ -0,0 +1,403 @@
+---
+title: SPIRE
+description: Describes how to configure Istio to integrate with SPIRE to get cryptographic identities through Envoy's SDS API.
+weight: 31
+keywords: [kubernetes,spiffe,spire]
+aliases:
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+[SPIRE](https://spiffe.io/docs/latest/spire-about/spire-concepts/) is a production-ready implementation of the SPIFFE specification that performs node and workload attestation in order to securely
+issue cryptographic identities to workloads running in heterogeneous environments. SPIRE can be configured as a source of cryptographic identities for Istio workloads through an integration with
+[Envoy's SDS API](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret). Istio can detect the existence of a UNIX Domain Socket that implements the Envoy SDS API on a defined
+socket path, allowing Envoy to communicate and fetch identities directly from it.
+
+This integration with SPIRE provides flexible attestation options not available with the default Istio identity management while harnessing Istio's powerful service management.
+For example, SPIRE's plugin architecture enables diverse workload attestation options beyond the Kubernetes namespace and service account attestation offered by Istio.
+SPIRE's node attestation extends attestation to the physical or virtual hardware on which workloads run.
+
```

Comment:
Let us make sure we add a warning that Istio 1.14 is required, assuming this feature lands in 1.14.

Reply:
I am not sure I agree? This is the docs for Istio 1.14 - it should be assumed everything needs 1.14?

Reply:
Thinking more about it, i think it would be useful to note both data plane and control plane are required to be 1.14.  (Is this correct?)

Reply:
Only would be useful to clarify if this only works with NEW istio 1.14 vs upgrading to Istio 1.14 @alexandrealvino

Reply:
Hi @linsun, only the data plane and `istioctl` are required to be 1.14. I will clarify that on the docs.

Reply:
That  sounds great, thanks @alexandrealvino !